packages/dara-core/tests/python/test_derived_data_variable.py

```diff
@@ -343,7 +343,9 @@ def func(a: int, data=TEST_DATA):
             json={'filters': {'column': 'col1', 'value': 2}, 'cache_key': cache_key, 'ws_channel': 'test_channel'},
             headers=AUTH_HEADERS,
         )
-        assert data_second_response.json() == df_convert_to_internal(unfiltered[unfiltered['col1'] == 2]).to_dict(orient='records')
+        assert data_second_response.json() == df_convert_to_internal(unfiltered[unfiltered['col1'] == 2]).to_dict(
```

Comment:
Do we not have a test making a call for the schema with a cache key? It would be nice if a test caught an issue like that.

Reply:
Right, totally forgot about DDVs and that they use a different store. Added implementation and checked those in the test.